docs/serving/using-a-custom-domain.md

```diff
@@ -1,11 +1,16 @@
-# Setting up a custom domain
+# Setting up a custom domain template
```

Comment:
I'm wondering if a more direct name for this might be something like one of the following:
- Changing the default domain
- Assigning a default domain